client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -21,19 +22,22 @@ class ConversationCaterpillarComponent extends React.Component {
 
 	render() {
 		const { comments, translate } = this.props;
-		const lastComment = last( comments );
-		const lastCommenterName = get( lastComment, 'author.name' );
-		const commentCount = comments.length;
+		const commentCount = size( comments );
+
+		// Only display authors with a gravatar, and only display each author once
+		const uniqueAuthors = uniqBy( map( comments, 'author' ), 'ID' );
+		const displayedAuthors = filter( uniqueAuthors, author => !! author.avatar_URL );
```

Comment:
nittiest nit and not even better, but this is cool: `const displayAuthors = reject( uniqueAuthors, 'avatar_URL' );`